st2common/bin/st2-generate-schemas

```diff
@@ -26,13 +26,15 @@ from st2common.models.api import action as action_models
 from st2common.models.api import pack as pack_models
 from st2common.models.api import policy as policy_models
 from st2common.models.api import rule as rule_models
+from st2common.models.api import sensor as sensor_models
 
 content_models = {
     'pack': pack_models.PackAPI,
     'action': action_models.ActionAPI,
     'alias': action_models.ActionAliasAPI,
     'policy': policy_models.PolicyAPI,
     'rule': rule_models.RuleAPI,
+    'sensor': sensor_models.SensorTypeAPI,
 }
 
 def main():
```

Comment:
I know that's an existing code, but still - https://github.com/StackStorm/st2/pull/5200/files#diff-2ac436102029c3245a263895b2bb3a5639f7d52c8396aefdeb0500deda58e23dR41.

Rather "interesting" way to get current file, ``__file_`` is much more commend and used elsewhere in the code base (``BASE_DIR = os.path.dirname(os.path.abspath(__file__))``).

Reply:
Another quick comment on an existing line - https://github.com/StackStorm/st2/blob/fcfed7c72afa728fc654bdd02a9a1623d7ed9434/st2common/bin/st2-generate-schemas#L48

I know ``name`` is controlled by us in this case, but as a general secure coding practice, we should still use ``os.path.join()`` to avoid potential undesired path manipulation (if the code ever gets changed so the name is an input or similar), so ``        schema_file = os.path.join(schemas_dir, "%s.json" % (name))``.

Reply:
Fixed.

Reply:
Fixed.